mysql-test/suite/versioning/t/federated.test

```diff
@@ -0,0 +1,66 @@
+--source include/not_embedded.inc
+--source include/have_partition.inc
+--source suite/versioning/engines.inc
+--source suite/versioning/common.inc
+--source suite/versioning/key_type.inc
+
+install soname 'ha_federatedx';
+
```

Comment:
* better put the test versioning into federated suite, not vice versa.
* make sure it is correctly skipped if federatedx is not built